### PR TITLE
MGMT-10345 [ZTPFW] Change Error messages by warn

### LINKED ISSUE #302: information message with "Error" in the text
we should consider to remove the word "Error" from informational print statements

### LINKED ISSUE #306: remove `wget` command
remove `wget` command from all bootstrap / scripts and replaces it with a standard command such as `curl`

### PR DESCRIPTION
**Pull request by eifrach**
Signed-off-by: Eran Ifrach <eifrach@redhat.com>

# Description

* Change all informational message from `ERROR` to `INFO` 
FIX #302

 * change wget to curl
 FIX #306

Fixes # (issue)

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: deploy-disconnected-registry/olm-sync.sh [hunk 1]
```diff
@@ -121,8 +121,8 @@ function mirror() {
             echo ">>>> Pruning index image finished: ${OLM_DESTINATION_INDEX}"
             retry=0
         else
-            echo ">>>> ERROR: Pruning index image: ${OLM_DESTINATION_INDEX}"
-            echo ">>>> ERROR: Retrying in 10 seconds"
+            echo ">>>> INFO: Failed runing index image: ${OLM_DESTINATION_INDEX}"
```
Comment (iranzo) [inline deploy-disconnected-registry/olm-sync.sh:124]: missing 'p' in pruning

**Review by iranzo — CHANGES_REQUESTED** [review]
lgtm, just the nit

**Review by iranzo — APPROVED** [review]
LGTM!

**Comment by alknopfler** [comment]
@eifrach could you test ci? after updating branch? I don't know why first time test didn't launch, but if not, just switch off/on the test-ci label

**Pull request merged**